lms/djangoapps/grades/api/tests/test_views.py

```diff
@@ -128,7 +130,10 @@ def get_url(self, username):
                 'course_id': self.course_key,
             }
         )
-        return "{0}?username={1}".format(base_url, username)
+        if username:
+            return "{0}?username={1}".format(base_url, username)
```

Comment:
@pomegranited nit: might make sense to use `urlencode` even though usernames are usually free of non-URL-friendly characters